java/avro/pom.xml

```diff
@@ -41,22 +41,17 @@
     <dependency>
       <groupId>org.apache.kafka</groupId>
       <artifactId>kafka-clients</artifactId>
-      <version>2.3.1</version>
+      <version>3.2.0</version>
```

Comment:
do we need to increment this dependency because of serialization interface? we usually want to support as low version as we can.